src/config.ts

```diff
@@ -46,3 +46,5 @@ export const generateStorageKey = (
 export const FEED_SURVEY_INTERVAL = 30;
 
 export const WEBAPP_MAGIC_IMAGE_PREFIX = `/image-generator`;
+
+export const SQL_QUERIES_PATH = '/opt/app/queries';
```

Comment:
now it won't work locally? should we just add it inline and lint with https://github.com/dailydotdev/daily-api/pull/2406

Reply:
It works inside the container. But I'm going to adjust this to be better later, just want to get it out and test it.